libs/common/src/lib/widgets/widget-generator.component.ts

```diff
@@ -55,6 +55,7 @@ export class WidgetGeneratorComponent implements OnInit, OnDestroy {
       map((account) => account?.type),
     ),
   ]).pipe(map(([featureEnabled, accountType]) => featureEnabled && accountType === 'stash-business'));
+  canHideLogo = this.sdk.currentAccount.pipe(map((account) => account.type !== 'stash-trial'));
```

Comment:
actually, product team has just asked a change about it: it must be for 'stash-startup', 'stash-growth' and 'stash-enterprise'.